docs/running-on-yarn.md

```diff
@@ -60,6 +60,14 @@ Running Spark on YARN requires a binary distribution of Spark which is built wit
 Binary distributions can be downloaded from the [downloads page](http://spark.apache.org/downloads.html) of the project website.
 To build Spark yourself, refer to [Building Spark](building-spark.html).
 
+To make Spark runtime jars accessible from YARN side, basically you will have 3 options:
```

Comment:
This section is mostly redundant with the configuration table. You should instead just redirect the user to read that, if you really want to add something.